packages/runner-shared/src/iframe/aut-iframe.js

```diff
@@ -69,6 +69,39 @@ export class AutIframe {
     return Cypress.cy.detachDom(this._contents())
   }
 
+  /**
+   * If the AUT is cross origin relative to top, a security error is thrown and the method returns false
+   * If the AUT is cross origin relative to top and chromeWebSecurity is false, origins of the AUT and top need to be compared and returns false
+   * Otherwise, if top and the AUT match origins, the method returns true.
+   * If the AUT origin is 'null', that means the src attribute has been stripped off the iframe and is adhering to same origin policy
+   */
+  doesAUTMatchTopOriginPolicy = () => {
+    const Cypress = eventManager.getCypress()
+
+    if (!Cypress) return
+
+    try {
+      const { location: locationAUT } = this.$iframe[0].contentWindow.document
+      const locationTop = Cypress.utils.locExisting(window.location.href)
+
+      return locationTop.origin === locationAUT.origin || locationAUT.origin === 'null'
+    } catch (err) {
+      if (err.name === 'SecurityError') {
+        return false
+      }
+
+      throw err
+    }
+  }
+
+  /**
+   * Removes the src attribute from the AUT iframe, resulting in 'about:blank' being loaded into the iframe
+   * See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-src for more details
+   */
+  removeSrcAttributeFromAUTIframe = () => {
```

Comment:
```suggestion
  removeSrcAttribute = () => {
```

Just a nitpick, but it seems implied that `autIframe.removeSrcAttribute` will remove it from the AUT iframe.

Reply:
makes sense IMO. Updated in [5c0588a](https://github.com/cypress-io/cypress/pull/20949/commits/5c0588a00f98865933cbb9fa911bf1e71738a544)